doc/articles/features/ElevatedView.md

```diff
@@ -0,0 +1,49 @@
+# ElevatedView
+On many design languages like _Material Design_, there's a notion of elevation where a portion of the UI should be presented has been _elevated_ over the rest of the content.
+
+In this case, the UWP proposed solution can't work on all platforms because of technical limitations. To address this problem, Uno provides a control called `ElevatedView`, able to produce a similar result on all platforms, providing an elevated result.
```

Comment:
```suggestion
In this case, UWP's native elevation API can't work on all platforms because of technical limitations. To address this problem, Uno provides a control called `ElevatedView`, able to produce a similar elevated effect on all platforms (UWP, Android, iOS, WebAssembly, and macOS).
```